<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>properties</class>
 <widget class="QDialog" name="properties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Properties</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>15</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tab_widget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="basic">
      <attribute name="title">
       <string>Basic</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="layout_3">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="layout">
           <item>
            <widget class="QLabel" name="file_icon">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>120</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>120</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="layout_4">
           <property name="spacing">
            <number>10</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="layout_5">
             <property name="spacing">
              <number>8</number>
             </property>
             <item>
              <widget class="QLabel" name="file_name_label">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>File Name:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="file_name">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="layout_6">
             <property name="spacing">
              <number>8</number>
             </property>
             <item>
              <widget class="QLabel" name="file_size_label">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Size:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="file_size">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="spacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layout_7">
         <property name="spacing">
          <number>8</number>
         </property>
         <item>
          <widget class="QLabel" name="accessed_label">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Accessed:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="accessed">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layout_8">
         <property name="spacing">
          <number>8</number>
         </property>
         <item>
          <widget class="QLabel" name="location_label">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Location:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="location">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="spacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="sequence">
      <attribute name="title">
       <string>Sequence</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="layout_9">
         <property name="spacing">
          <number>8</number>
         </property>
         <item>
          <widget class="QLabel" name="name_label">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="name">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layout_10">
         <property name="spacing">
          <number>8</number>
         </property>
         <item>
          <widget class="QLabel" name="owner_label">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Owner:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="owner">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layout_11">
         <property name="spacing">
          <number>8</number>
         </property>
         <item>
          <widget class="QLabel" name="status_label">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Status:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="status">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layout_12">
         <property name="spacing">
          <number>8</number>
         </property>
         <item>
          <widget class="QLabel" name="reader_id_label">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Reader ID:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="reader_id">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layout_13">
         <property name="spacing">
          <number>8</number>
         </property>
         <item>
          <widget class="QLabel" name="amount_label">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Samples amount:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="amount">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layout_19">
         <property name="spacing">
          <number>8</number>
         </property>
         <item>
          <widget class="QLabel" name="protocol_label">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Protocol:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="protocol">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layout_14">
         <property name="spacing">
          <number>8</number>
         </property>
         <item>
          <widget class="QLabel" name="creaton_date_label">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Creation date:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="creaton_date">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layout_15">
         <property name="spacing">
          <number>8</number>
         </property>
         <item>
          <widget class="QLabel" name="last_modification_label">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Last modification:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="last_modification">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layout_16">
         <property name="spacing">
          <number>8</number>
         </property>
         <item>
          <widget class="QLabel" name="nitrogen_use_label">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Nitrogen use:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="nitrogen_use">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layout_17">
         <property name="spacing">
          <number>8</number>
         </property>
         <item>
          <widget class="QLabel" name="dose_rate_label">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Dose rate:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="dose_rate">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layout_18">
         <property name="spacing">
          <number>8</number>
         </property>
         <item>
          <widget class="QLabel" name="external_dose_rate_label">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>External dose rate:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="external_dose_rate">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttons_area">
     <item>
      <spacer name="spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="push_button_close">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tab_widget</tabstop>
  <tabstop>push_button_close</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
